### DISCUSSION TITLE
self-signed certificate in certificate chain

### DISCUSSION THREAD

**Discussion by mahadea** [zowe/zowe-cli#1634] [Q&A] [answered]
Hi,
I am trying to connect to a zOS host via z/osmf from VS Code. I got the zowe profile in place and everything and when I issue this command
zowe zosmf check status --host <hostname> --user <userid> --pass <password>
Command Error:
z/OSMF REST API Error:
http(s) request error event called
**Error: self-signed certificate in certificate chain**
Error Details:
HTTP(S) client encountered an error. Request could not be initiated to host.
Review connection details (host, port) and ensure correctness.

Protocol:  https
Host:      <hostname>
Port:      443
Base Path:
Resource:  /zosmf/info
Request:   GET
Headers:   [{"X-CSRF-ZOSMF-HEADER":true}]
Payload:   undefined

What am I doing wrong?

Thanks

**Comment by zFernand0** [comment, marked as answer]
It seems like this is a question regarding the use of Zowe CLI.
I believe that you could use set the `rejectUnauthorized` property to false in your `zowe.config.json` files 😋 

Here is a link that might help a little bit.
Source: https://docs.zowe.org/stable/user-guide/cli-using-working-certificates
